root-packages/runc/build.sh

```diff
@@ -6,17 +6,62 @@ TERMUX_PKG_VERSION="1.1.14"
 TERMUX_PKG_SRCURL=https://github.com/opencontainers/runc/archive/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=563cf57c38d2e7149234dbe6f63ca0751eb55ef8f586ed12a543dedc1aceba68
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="libseccomp"
+TERMUX_PKG_BUILD_DEPENDS="libseccomp-static"
 
 termux_step_make() {
+	cat > fakes.c << EOF
```

Comment:
This should probably be a separate `fakes.c.patch`
unless you have a good reason it needs to be inlined.

Reply:
There is even no need in `fakes.c.patch`. It can be stored as simple `fakes.c`. And I do not see a reason to pick regular `gcc` or `ar`, they are exported at termux environment configuring stage.

Reply:
fixed